### PR TITLE
Sc94150/update schema

### PR DESCRIPTION
**Pull request by leah-is-offline**
### Ticket

[sc94150]

### Changes made

- modify post request to send a pathName and displayName to remote endpoint
 
### Related PRs

- waiting on https://github.com/neofinancial/coverage-upload-action/pull/18 in this repository

### Release Plan

- half of this ticket involves work on the service providing the remote endpoint. I am planning on finishing the other half of this ticket in [sc92910] on the service.
- The plan is to merge this PR into [sc95777] and then merge [sc95777] into [sc92910].

### Checklist

- [x] Have you fully tested your PR locally

### CODE REVIEW

**Comment by neo-anand-patel** [comment]
I'll do the other half of this ticket to save the `displayName` and `pathName` on [this](https://github.com/neofinancial/coverage-dashboard/pull/67) PR.

**Review by AD-Blue — APPROVED** [review]

**Review by AdamTranquilla — APPROVED** [review]
Looks good.

NIT: you could use more descriptive commit messages when addressing changes. that way if you want to revert a specific change you can identify it quickly

**Review by PeterHuang516 — APPROVED** [review]

**Review by leah-is-offline — COMMENTED** [review]
approve! but i can approve because it says I opened

**Review by gate3 — APPROVED** [review]

File: src/get-data.ts [hunk 1]
```diff
@@ -5,13 +5,15 @@ import getCoverage from './get-coverage';
...
 
-const getData = async (pathway: string): Promise<PRData> => {
+const getData = async (pathway: string, displayName: string): Promise<PRData> => {
   const authToken = getInput('coverageToken');
 
   const prData: PRData = {
     repositoryId: context.payload.repository?.id,
     ref: '',
     baseRef: '',
+    pathway,
```
Comment (iansu) [inline src/get-data.ts:15]: This should just be `path`

**Review by iansu — CHANGES_REQUESTED** [review]
You're going to need to update the CHANGELOG and the version in package.json. We also need to know if this is going to be a breaking change.

**Pull request closed without merging**